releasenotes/notes/add-litinski-transformation-ec42e98cc378613f.yaml

```diff
@@ -0,0 +1,40 @@
+---
+features_transpiler:
+  - |
+    Added a new transpiler pass :class:`qiskit.transpiler.passes.LitinskiTransformation`
+    that implements the transform described in `arXiv:1808.02892 <https://arxiv.org/abs/1808.02892>`__.
+
+    The input to the pass is a circuit with Clifford and single-qubit RZ-rotation gates, and the output
+    is a circuit with multi-qubit Pauli rotations (implemented as :class:`.PauliEvolutionGate` gates)
+    followed by Clifford gates. 
+
+    The pass supports all of the Clifford gates in the list returned by :func:`.get_clifford_gate_names`,
+    namely
+    ``["id", "x", "y", "z", "h", "s", "sdg", "sx", "sxdg", "cx", "cz", "cy", "swap", "iswap", "ecr", "dcx"]``.
+    The list of supported RZ-rotations is ``["t", "tdg", "rz"]`` (we automatically convert T and Tdg gates
+    to RZ-rotations).
+
```

Comment:
This seems a bit overly specific for the reno -- they are not the docs 🙂 
```suggestion
```

Reply:
I actually think that we want to be specific and explain which gates are supported by the pass. So instead of removing the informations from the release notes, I have also added it to the pass description: [bc76da4](https://github.com/Qiskit/qiskit/pull/14753/commits/bc76da40133040bc8ec01fd8d315dac8a030bdce).

BTW, look at the release notes for one of Matthew's PRs: https://github.com/Qiskit/qiskit/pull/14803/.